libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -934,16 +934,21 @@ void ofFbo::updateTexture(int attachmentPoint) {
 			glPushAttrib(GL_COLOR_BUFFER_BIT);
 		}
 
-		bind();
+		// Because we temporarily change global Fbo binding state
+		// when blitting from one framebuffer to the other, we first have 
+		// to query the renderer for its binding state, and store it.
+		GLuint rendererFboBindingState = ofGetGLRenderer()->getCurrentFramebufferId();
```

Comment:
oh, i see, man opengl state is such a pain... i've refactored this a bit, it's doing the same as your solution but the renderer is now keeping track of everything. if you can check that i'm not doing something wrong:

https://github.com/openframeworks/openFrameworks/compare/master...arturoc:refactor-fbo?expand=1